onnxruntime/core/providers/coreml/mlprogram_test_scripts/DebugMLProgram.md

```diff
@@ -25,6 +25,8 @@ https://apple.github.io/coremltools/docs-guides/source/model-intermediate-langua
 Usage is reasonably intuitive. The below example defines a model with 2 inputs and a matmul operator.
 The model is printed, and run with randomly generated inputs. The output from doing so is printed.
 
+There are additional test scripts in this directory for different operators.
```

Comment:
are these scripts just to show what's going on or will they eventually be comparing the output of direct CoreML with the CoreML EP?

Reply:
Intended as scripts that can be used to validate changes/debug issues. Mainly for operators where I've had issues getting the output to match (otherwise I wouldn't have created the script). 

They're also provide a reference to create a new script as there are a few different things that might be needed (constants, variadic inputs, multiple steps, NN model) that are spread across them. I don't think we should need too many more though.